source/common/http/conn_pool_grid.cc

```diff
@@ -325,13 +325,13 @@ ConnectionPool::Cancellable* ConnectivityGrid::newStream(Http::ResponseDecoder&
   } else {
     // Before skipping to the next pool, make sure it has been created.
```

Comment:
I think we might want to tweak this comment since "skipping to the next pool" is a tag less clear than "using the h2 pool"?

Reply:
ping